### ISSUE TITLE
Support for Internxt

### ISSUE THREAD

**Issue by claell** [rclone/rclone#7610]
#### The associated forum post URL from `https://forum.rclone.org`
https://forum.rclone.org/t/internxt-rclone-support/39952/1

#### What is your current rclone version (output from `rclone version`)?
Irrelevant

#### What problem are you are trying to solve?
Support for another storage provider.

#### How do you think rclone should be changed to solve that?
Integrate a connector.

####
Additional information:
* Internxt JS SDK: https://github.com/internxt/sdk
* Some API (might not be offering the required features, yet and seems to be WIP): https://api.internxt.com/drive/

#### How to use GitHub

* Please use the 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to show that you are affected by the same issue.
* Please don't comment if you have no relevant information to add. It's just extra noise for everyone subscribed to this issue.
* Subscribe to receive notifications on status change and new comments.

**Comment by raffaelevall** [comment]
Hi, webdav with cli, for data encryption, has been released last month. 
Maybe now it's easier to integrate a connector. 
I'm not an expert about rclone, but I managed to connect the internxt webdav and mount it but both the `rclone copy` and the `rclone sync `receive errors.

**Comment by progressify** [comment]
subscribed

**Comment by claell** [comment]
Also see https://www.reddit.com/r/internxt/comments/1etvp3r/internxt_now_supports_rclone/ (not native support, but through Internxt CLI and WebDAV).

The Internxt support also told me that they were working on native support. Though not sure about that. Let's see, whether a dev from them will join this issue.

**Comment by scottnzuk** [comment]
guys i found out why it will never work.. internxt don't really have webdav support.
https://github.com/internxt/cli/issues/248

they don't support two most critial features need. Copy function being one of them almost laughable. im not sure if the wizards at RCLONE can make a profile that is not use copy or proppatch.


COPY        | ❌ | Enables in-place duplication, versioning, efficient syncing
PROPPATCH | ❌ | Enables timestamp and metadata preservation

**Comment by scottnzuk** [comment]
You’re correct in noting that Internxt’s WebDAV implementation currently lacks support for the COPY method, which is a standard operation in the WebDAV protocol used to duplicate files or directories on the server. This limitation can indeed hinder certain workflows that rely on this functionality. AKA ALL BACKUP SYSTEM LOL

**Internxt’s WebDAV feature is designed with a strong emphasis on client-side encryption and privacy. The WebDAV server operates locally on your machine via the Internxt CLI, ensuring that your data is encrypted before transmission and decrypted only on your device. While this architecture enhances security, it also means that some standard WebDAV operations, like COPY, are not implemented.  ￼ ￼ ￼**

For tasks requiring file duplication, Internxt recommends using their CLI commands. For example, to move files, you can use:

internxt move-file <source> <destination>
internxt move-folder <source> <destination>

However, there isn’t a direct CLI command for copying files. As a workaround, you can manually download the file and re-upload it to the desired location:

internxt download-file <source>
internxt upload-file <destination>

This process effectively creates a copy of the file in the new location.

If the absence of the COPY method is a significant obstacle for your workflow, you might consider reaching out to Internxt’s support team to inquire about potential future support for this feature.

For more detailed information on Internxt’s WebDAV capabilities and limitations, you can refer to their Help Center:  ￼

**Comment by scottnzuk** [comment]
rclone with --webdav-method PUT and modtime disabled if this is a thing (chatgpt could be hallucination) 

to make a profile for internxt?

**Comment by Neustradamus** [comment]
Dear all,

Can you look this PR?
- https://github.com/rclone/rclone/pull/9110
